lib/ex2ms.ex

```diff
@@ -133,24 +133,29 @@ defmodule Ex2ms do
     [translate_cond(expr, state)]
   end
 
+  @deprecated "^ is no longed used in match conditions and bodies"
```

Comment:
This won't do anything since it's a private function. Instead leave a note that we should deprecate it in the future by printing a message to the user.

Reply:
Good catch, didn't expect that. Fixed:

```
warning: ^ is no longed used in match conditions and bodies
  test/ex2ms_test.exs:286: Ex2msTest."test deprecated syntax for composite bound variables in return value"/1
```